### PR TITLE
To Do list: add & remove

### PR DESCRIPTION
**Pull request by billymohajeri**
In this step I,
- Removed all hardcoded items from the tasks array.
- Created a new JavaScript file for the new functionality.
- Implemented a function for adding a new task.
- Implemented a function for deleting a task.
- Implemented a function for editing task descriptions.
- Saved changes in local storage.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -46,14 +66,81 @@ window.addEventListener('load', () => {
...
+    editTextBox.focus();
+  }
+
+  // Save button ==================
+  if (e.target.className === 'fa-regular fa-floppy-disk fa-xs') {
+    saveFunc(e);
+  }
+});
+
+const clearList = document.querySelector('.clear');
```
Comment (Ismailco) [inline src/index.js:141]: - [ ] When I click `clear all completed` all the tasks got removed from the array, kindly make sure to fix this so only the completed tasks get deleted

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @billymohajeri, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
**- Adding clear instructions on how to use the app so users know how this app should be used (is there a command to start the app? should you build the app before using it? ...)**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (Abenezer-Tilahun) [reply src/index.js:141]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @billymohajeri 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct GitHub-Flow.
✔️ Professional README  
✔️ Good Commit Messages


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**